src/dev-app/BUILD.bazel

```diff
@@ -121,11 +121,11 @@ create_system_config(
     output_name = "system-config.js",
 )
 
```

Comment:
@josephperrott I don't know how to resolve the lint errors in this file. I get a lint failure even after I run `yarn format` and the error doesn't actually say what's wrong. Here's the full log:

```
yarn run v1.22.10
Checking format of 1 file(s)

The following files are out of format:
    - src/dev-app/BUILD.bazel

? Format the files now? Yes
Formatting 1 file(s)
[========================================] ETA: 0s | 1/1 files
√  Formatting complete.
Done in 5.56s.
```

Reply:
This issue was actually just reported a few days ago in another case.  I need to address the error messaging that occurs to surface it.

For the specific situation, you can have a glob without a matcher.

You should instead have


```
filegroup(
    name = "variables",
    srcs = [
        "variables.json"
    ],
)
```

Reply:
The problem is that the `variables.json` is an optional file that is `.gitignore`d. Wouldn't using a `filegroup` like this result in an error if the file doesn't exist?

Reply:
I think I ran into this problem too. You need the glob, because the file doesn't necessarily exist, but it needs some arbitrary matcher because of the lint issue, I assume due to the assumption that you would never need a glob for a single file.

Reply:
Nice, adding a `*` in there fixed it 👍